publisher-sdk/src/main/java/com/criteo/publisher/util/AndroidUtil.java

```diff
@@ -57,4 +63,25 @@ public int dpToPixel(int dp) {
     return (int) Math.ceil(dp * context.getResources().getDisplayMetrics().density);
   }
 
+  /**
+   * Check whether the current process is in foreground.
+   *
+   * @return {@code true} if the current process is in foreground, {@code false} otherwise
+   */
+  public boolean isCurrentProcessInForeground(@NonNull Context context) {
```

Comment:
Do you know a way to test this ? It would be great to cover such kind of low level code

Reply:
I will try to cover this using https://developer.android.com/reference/androidx/test/rule/ServiceTestRule